api/src/controllers/changes.js

```diff
@@ -495,7 +541,12 @@ module.exports = {
           }
           res.type('json');
           defibrillator(feed);
-          getChanges(feed);
+          if (feed.upstreamSessions.length) {
+            feed.log('initFeed().callback', 'this feed already has a getChanges() in progress; returning.');
```

Comment:
How could this ever be the case? We've only just created the feed object.

Reply:
Looks like you're right - feed is added to `continuousFeeds` directly above this, so we shouldn't have a race here.